ehrql/query_engines/csv.py

```diff
@@ -32,9 +31,21 @@ def get_results(self, variable_definitions):
         return super().get_results(variable_definitions)
 
     def populate_database(self, table_nodes):
-        # Populate the database using CSV files in the supplied directory
-        orm_classes = orm_classes_from_tables(table_nodes)
-        input_data = read_orm_models_from_csv_directory(
-            Path(self.csv_directory), orm_classes.values()
+        # Populate the database using the files in the supplied directory
+        self.database.setup(self.get_input_data(table_nodes))
+
+    def get_input_data(self, table_nodes):
```

Comment:
Does this belong in `orm_utils`?

Reply:
Possibly, but `orm_utils` is actually on my list to get rid of – or at least, to move into `tests/lib`. The ORM snuck in to production via a set of convenient-at-the-time changes but there's actually no need for it and it overcomplicates things. One thing at a time though!